views/js/randomColorsForScores.js

```diff
@@ -0,0 +1,18 @@
+window.onload = function () {
+  document.querySelectorAll("#high_scores_container .individual_score").forEach(function (element) {
+    var minLightnessRatio = 0.25;
+    var red = getRandomArbitrary(minLightnessRatio, 1);
+    var green = getRandomArbitrary(minLightnessRatio, 1);
+    var blue = getRandomArbitrary(minLightnessRatio, 1);
+    // scale colors to max of 255
```

Comment:
HSL or HSV (hue, saturation, brightness) might be a simpler model here if you're looking for fairly consistent brightness and high saturation, but random color.

Reply:
Done https://github.com/knative-portability/Kubercade/pull/17/commits/c74b0b10e985cb63751942b36b23a145d63e56ff, looks like https://screenshot.googleplex.com/6QkspTMD29c